announcing-pydoc-io.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: Nov 4, 2016
+   :tags: pydoc, documentation, api, reference
+
+Announcing pydoc.io beta
+========================
+
+Today we're excited to announce our latest project: https://pydoc.io.
+This work was made possible by the :doc:`MOSS Grant </rtd-awarded-mozilla-open-source-support-grant>`_ from Mozilla.
+Thanks to Mozilla for giving us the money to set aside time to build this wonderful community resource.
```

Comment:
Suggestion for clarity: `giving us money to set aside time to build` -> `funding our time building`